content/xp-arch-framework/interface-integrations/platform-continuity.md

```diff
@@ -0,0 +1,155 @@
+
+```yaml
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: backup-with-data
+  namespace: velero
+spec:
+  includedNamespaces:
+  - '*'
+  excludedNamespaces:
+  - kube-node-lease
+  - kube-system
+  - kube-public
+  - velero
+  excludedResources:
+  - bindings
+  - componentstatuses
+  - endpoints
+  - events
+  - limitranges
+  - nodes
+  - persistentvolumeclaims
+  - persistentvolume
+  - pods
+  - podtemplates
+  - replicationcontrollers
+  - resourcequotas
+  - services
+  - daemonsets
+  - deployments
+  - replicasets
+  - statefulsets
+  - cronjobs
+  - jobs
+  includeClusterResources: true
+  snapshotVolumes: false
+  metadata: {}
+  ttl: 720h0m0s
+ ```
+
+ We recommend configuring Velero to store the captured backup state for all your control planes in a single, global bucket or blob.
+
+{{< hint "tip" >}}
+Instead of excluding all other resources, you could include only the resources that you want to backup. However, this would require you to list _all_ resources types--including the CRDs introduced by providers--which isn't possible/feasible. Hence, we recommend doing it the way we show above.
+{{< /hint >}}
+
+### Backup Schedules
+
+We recommend establishing a Velero schedule to periodically take snapshots for each control plane. THe following is a sample Velero schedule object that illustrates a recommended schedule:
+
+```yaml
+apiVersion: velero.io/v1
+kind: Schedule
+metadata:
+  name: daily-backup
+  namespace: velero
+spec:
+  schedule: '@every 24h'
+  template:
+    includedNamespaces:
+    - '*'
+    excludedNamespaces:
+    - kube-node-lease
+    - kube-system
+    - kube-public
+    - velero
+    excludedResources:
+    - bindings
+    - componentstatuses
+    - endpoints
+    - events
+    - limitranges
+    - nodes
+    - persistentvolumeclaims
+    - persistentvolume
+    - pods
+    - podtemplates
+    - replicationcontrollers
+    - resourcequotas
+    - services
+    - daemonsets
+    - deployments
+    - replicasets
+    - statefulsets
+    - cronjobs
+    - jobs
+    includeClusterResources: true
+    snapshotVolumes: false
+    metadata: {}
+    ttl: 168h0m0s
+```
+
+For each control plane that you create, create three Velero `Schedule` objects illustrated above with the following schedules:
+
+- Hourly
+- Daily
+- Weekly
+
+We think this is a good baseline, but you can customize the frequency of the backups according to the Recovery Point Objective (RPO) for your platform. If you need to be able to restore state more frequently, you will want to make the snapshots more frequent than hourly.
+
+### Restore Control Plane from state
+
+To restore backup state into a new cluster:
+
+1. Ensure that Crossplane and providers aren't running in a new cluster. Otherwise, the ordering between managed resources, composites, and claims becomes critical. Failure to do so may result in race conditions and, for example, duplication of managed resources.
```

Comment:
> Ensure that Crossplane and providers aren't running in a new cluster.

If my backup contains my provider definitions, how do I control their scaling? Is there a field I need to override in my backups or during my restore?

Reply:
`pods` and `deployments` added to  `excludedResources` in velero backup definition - so you only need to check that your new target cluster is not running crossplane or any providers - scale to 0 or add replicas 0 in `controllerConfig` - otherwise folks need to check the backup manifests before restore